GiniSwitchSDK/GiniSwitchSDK/Classes/Authenticator.swift

```diff
@@ -52,10 +52,7 @@ class Authenticator {
 
         var authHeaders = basicAuthHeadersDictFor(user: clientId!, pass: clientSecret!)
         authHeaders["Accept"] = "application/json"      // TODO: don't hardcode
-        return Resource<Token>(url: fullUrl, headers: authHeaders, method: "GET", body: nil, parseJSON: { json in
-            guard let dictionary = json as? JSONDictionary else { return nil }
-            return Token(dictionary)
-        })
+        return Resource<Token>(url: fullUrl, headers: authHeaders, method: "GET", body: nil)
```

Comment:
I suggest you to create a simple `enum` with the HTTP methods, it is more clean and less prone to errors.